full/src/test/java/apoc/ttl/TTLClusterTest.java

```diff
@@ -0,0 +1,100 @@
+package apoc.ttl;
+
+import apoc.util.*;
+import org.junit.*;
+import org.neo4j.driver.Session;
+import org.neo4j.driver.SessionConfig;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import static apoc.util.TestUtil.isTravis;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assume.assumeFalse;
+
+
+public class TTLClusterTest {
```

Comment:
In realtà ho ragionato che non serve un cluster. Basta un istanza singola di Neo4j e poi vai di multi database